tools/autodeployment/deploy.sh

```diff
@@ -44,7 +44,7 @@ echo "Deleting $OLD Volumes"
 docker volume prune -f
 
 echo "Starting $ENV Environment"
-docker-compose --env-file $ENV_FILE --project-name=$ENV up -d
+GITHUB_TOKEN=$3 docker-compose --env-file $ENV_FILE --project-name=$ENV up -d
```

Comment:
I do not know the reason why you use `GITHUB_TOKEN=$3` instead of `GITHUB_TOKEN=$GITHUB_TOKEN` you implemented in previous commit?

Reply:
Because I forgot some basic things that were taught to us in our first year Linux class. I pass the token as the third argument [here](https://github.com/Seneca-CDOT/telescope/blob/62435161a7e06d55f4c08e414d10b4d7889a9014/tools/autodeployment/builds.js#L83). So in order to access it in the script I use the Dollar sign syntax.